test-suite/output/UnivBinders.out

```diff
@@ -88,9 +98,10 @@ The command has indeed failed with message:
 This object does not support universe names.
 The command has indeed failed with message:
 Cannot enforce v < u because u < gU < gV < v
-Monomorphic bind_univs.mono = Type@{u}
-     : Type@{u+1}
-(* {u} |=  *)
+Monomorphic bind_univs.mono = 
+Type@{TestSuite.bind_univs.1}
+     : Type@{TestSuite.bind_univs.1+1}
+(* {TestSuite.bind_univs.1} |=  *)
```

Comment:
This should be `TestSuite.bind_univs.u`.

Reply:
Fixed by your patch, right?